src/keystone.nim

```diff
@@ -72,7 +72,7 @@ type
     MISSINGFEATURE,
     MNEMONICFAIL
 
-  KeystoneError* = object of Exception
+  KeystoneError* = object of CatchableError
```

Comment:
I wasn't aware of the difference between `Exception` and `CatchableError`. Thanks for the fix.

Reply:
`Exception` is deprecated in favor of `CatchableError` or `Defect` too